src/AppInstallerCommonCore/Deployment.cpp

```diff
@@ -322,4 +322,23 @@ namespace AppInstaller::Deployment
 
         return packages.begin() != packages.end();
     }
+
+    void RegisterPackage(
+        std::string_view packageFamilyName,
+        IProgressCallback& callback)
+    {
+        PartialPercentProgressCallback progress{ callback, 100 };
+
+        progress.SetRange(0, 100);

```

Comment:
PartialPercentProgressCallback  is not needed if the range is 0 - 100, Just use the callback directly (and you already did)